### ISSUE TITLE
Failure to push to Heroku

### ISSUE THREAD

**Issue by merubin** [ga-wdi-exercises/project2#764]
I created account, logged in created application now I want to push my code to Heroku, but it wont let me something about a public key.  Below are the comands and errors:

```
$ git push heroku master
The authenticity of host 'heroku.com (50.19.85.132)' can't be established.
RSA key fingerprint is SHA256:8tF0wX2WquK45aGKs/Bh1dKmBXH08vxUe0VCJJWOA/o.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'heroku.com,50.19.85.132' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
  Thu Oct 20 10:01:00 /users/MICHAELS_MAC/wdi/projects/project2/portfolio_management_system (master)
```

**Comment by merubin** [comment]
Here was the Heroku app info:

```
$ heroku login
Enter your Heroku credentials.
Email: mike@rubinsworld.com
Password (typing will be hidden):
Logged in as mike@rubinsworld.com
  Thu Oct 20 09:55:03 /users/MICHAELS_MAC/wdi/projects/project2/portfolio_management_system (master)
$ heroku create
Creating app... done, ⬢ warm-retreat-59302
https://warm-retreat-59302.herokuapp.com/ | https://git.heroku.com/warm-retreat-59302.git
  Thu Oct 20 09:56:17 /users/MICHAELS_MAC/wdi/projects/project2/portfolio_management_system (master)
```

**Comment by jshawl** [comment]
Did you rename your heroku app by chance?

**Comment by jshawl** [comment]
if so, you'll need to update the remote url as well

**Comment by merubin** [comment]
The Heroku said no _ characters so I used - when I created the name.  Is this what you mean?
Please ELI5
$ git remote -v
heroku  git@heroku.com:portfolio-management-system.git (fetch)
heroku  git@heroku.com:portfolio-management-system.git (push)
origin  git@github.com:merubin/portfolio_management_system.git (fetch)
origin  git@github.com:merubin/portfolio_management_system.git (push)
  Thu Oct 20 10:51:37 /users/MICHAELS_MAC/wdi/projects/project2/portfolio_management_system (master)
$

**Comment by jshawl** [comment]
ok that looks good! what is the output of `git push heroku master` now?

**Comment by merubin** [comment]
$ git push heroku master
Warning: Permanently added the RSA host key for IP address '50.19.85.154' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
  Thu Oct 20 11:00:20 /users/MICHAELS_MAC/wdi/projects/project2/portfolio_management_system (master)

**Comment by ghost** [comment]
Hi Mike. Have you authorized Github to connect with Heroku in the dashboard? That might help.

**Comment by merubin** [comment]
I thought I did that here is the screen print 
![image](https://cloud.githubusercontent.com/assets/19254862/19565565/3ccc3100-96b5-11e6-8966-c0b180dc1d89.png)

**Comment by merubin** [comment]
I believe it is something with the SSH key

**Comment by merubin** [comment]
Maybe I should revoke and recconnect?

**Comment by ghost** [comment]
Yes maybe. I know I authorized and was able to push afterwards.
